.github/workflows/shippable_builds.yml

```diff
@@ -572,6 +581,21 @@ jobs:
           echo "app_sha=$APP_SHA" >> $GITHUB_OUTPUT
           cat $GITHUB_OUTPUT
 
+      - name: Display GitHub Draft Notes
+        if: ${{ contains(matrix.releaseTarget, 'github') }}
+        shell: bash
+        env:
+          APPLICATION_LABEL: ${{ steps.pkginfo.outputs.APPLICATION_LABEL }}
+          FULL_VERSION_NAME: ${{ steps.pkginfo.outputs.FULL_VERSION_NAME }}
+          VERSION_CODE: ${{ steps.pkginfo.outputs.VERSION_CODE }}
+          THUNDERBIRD_GITHUB_NOTES: ${{ needs.release_commit.outputs.thunderbird_github_notes }}
+        run: |
+          if [[ "$APP_NAME" == "thunderbird" ]]; then
+            echo "<h2>${APPLICATION_LABEL} ${FULL_VERSION_NAME} GitHub Draft Release Notes (${VERSION_CODE})</h2><pre>" | tee $GITHUB_STEP_SUMMARY
+            echo "${THUNDERBIRD_GITHUB_NOTES}" | sed -z 's/\n\n/\n/g' | tee -a $GITHUB_STEP_SUMMARY
+            echo "</pre>" | tee -a $GITHUB_STEP_SUMMARY
+          fi
```

Comment:
For the step summary, let's rather show them early. We can do this together with the short form notes.

At this point in the process, we instead need to add them to the publish_gh step below instead so they get added to the github release.

Reply:
What you'll need to do is extend the Get release sha step to output `app_release_notes` based on the appname, and then use this in the publish_gh step.

Reply:
Ok. I wasn't sure if we were ready to publish the notes as they are rendered now, or if you wanted to just print them out for now and fine-tune manually.